README.md

```diff
@@ -23,6 +23,23 @@ This can be useful in several ways:
 
 `./idpbuilder create --buildName localdev`
 
+You can also define the kubernetes version to image and which corresponds to the kind pre-built [image](https://github.com/kubernetes-sigs/kind/releases).
+`./idpbuilder create --kubeVersion v1.27.3`
+
+If it is needed to expose some extra Ports between the docker container and the kubernetes host, they can be declared as such
+`./idpbuilder create --extraPorts 22:32222`
+
+It is also possible to use your own kind config file
+`./idpbuilder create --buildName local --kindConfig ./my-kind.cfg`
```

Comment:
> Should be `./my-kind.yaml`. Kind cluster configs are usually a yaml file I think.

Yes but it can be suffixed with `*.cfg` too ;-) I will rename it